portlet/vue-portlet/pom.xml

```diff
@@ -0,0 +1,18 @@
+<project>
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.acme.samples</groupId>
+	<artifactId>vue-sample</artifactId>
```

Comment:
Both portlets have the same artifacts. IMO we should make them unique to make sure there will be no conflict if both are built.
Same for the extension names.

Reply:
Done